cfgov/paying_for_college/documents.py

```diff
@@ -10,6 +12,7 @@ class SchoolDocument(Document):
 
     autocomplete = fields.TextField(analyzer=label_autocomplete)
     text = fields.TextField(attr='primary_alias', boost=10)
+    url = fields.TextField()
     nicknames = fields.TextField()
 
     def prepare_autocomplete(self, instance):
```

Comment:
Autocomplete is not catching nicknames or school IDs. I think we need to send all those values to the index as a string. Something like this:

```python
    def prepare_autocomplete(self, instance):
        alias_strings = [a.alias for a in instance.alias_set.all()]
        nickname_strings = [n.nickname for n in instance.nickname_set.all()]
        auto_strings = alias_strings + nickname_strings + [str(instance.pk)]
        return " ".join(auto_strings)
```

Note that I can't make this a code suggestion because I can't select the full method.

Reply:
I updated autocomplete to include school IDs.